api/models/site.js

```diff
@@ -165,6 +165,8 @@ module.exports = (sequelize, DataTypes) => {
       afterValidate,
       validationFailed,
     },
+    timestamps: true,
```

Comment:
I think the `timestamps` option defaults to `true`, right? Specifying it explicitly, while not really harming anything, makes it less clear that other models without it specified also have timestamps enabled.

Reply:
Yeah, I made it explicit because I was seeing weird errors i which sequelize would complain that it couldn't find the field 'deletedAt' on the models; but those errors might have been mitigated by adding the `sequelize.sync` call to our test bootstrap file. I'll try removing the `timestamps` declaration.